modules/io/common/src/main/java/org/locationtech/jts/io/geojson/TransformationUtils.java

```diff
@@ -0,0 +1,85 @@
+package org.locationtech.jts.io.geojson;
+
+import org.locationtech.jts.algorithm.Orientation;
+import org.locationtech.jts.geom.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TransformationUtils {
+
+    /**
+     * Transforms a geometry using the Right Hand Rule specifications defined
+     * in the latest GeoJSON specification.
+     * See <a href="https://tools.ietf.org/html/rfc7946#section-3.1.6">RFC-7946 Specification</a> for more context.
+     *
+     * @param geometry to be transformed
+     * @return Geometry under the Right Hand Rule specifications
+     */
+    public static Geometry enforceRightHandRuleOrientation(final Geometry geometry) {
```

Comment:
just call this `transformRightHandRule`.

Reply:
I followed this recommendation 👍